src/Customer/Service.php

```diff
@@ -17,4 +18,42 @@ interface Service
      * @return Customer
      */
     public function create(Token $token, string $email, string $description): Customer;
+
+    /**
+     * Retrieve a Customer.
+     *
+     * @param Token $token
+     *
+     * @return Customer
+     */
+    public function find(Token $token): Customer;
```

Comment:
Again, if the type of token doesn't have any affect (I just looked down and don't think it does) I think this should just accept the token value as a `string $uid` parameter?